### PR TITLE
adding large residual check

### PR DESCRIPTION
**Pull request by WallyMaier**
## Proposed Changes
This adds a check to prevent SU2 continuing diverge/reach extremely large residuals.  This seems to be a feature request in both the SU2 issues (#38) and on the CFD-Online forum.   

This is similiar to the divergence criteria but is activated at residuals greater than 10^12 . This number can change, it was arbitrarily chosen. 
 
- [ x] I am submitting my contribution to the develop branch.
- [x ] My contribution generates no new compiler warnings (try with the '-Wall -Wextra -Wno-unused-parameter -Wno-empty-body' compiler flags).
- [x ] My contribution is commented and consistent with SU2 style.
- [ ] I have added a test case that demonstrates my contribution, if necessary.

### CODE REVIEW

**Comment by pcarruscag** [comment]
Hi @WallyMaier,
In most of my cases the initial residual for SST is ~10 and sometimes it goes over 15 before starting to drop, probably a normalization thing... anyway, can you make it 20?
Cheers,
Pedro

**Comment by WallyMaier** [comment]
@pcarruscag done deal

File: SU2_CFD/src/solver_structure.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/*!
+﻿/*!
  * \file solver_structure.cpp
  * \brief Main subrotuines for solving direct, adjoint and linearized problems.
```
Comment (economon) [inline SU2_CFD/src/solver_structure.cpp]: Could you please fix that old typo for ‘subroutines’ in the \brief? Also, we don’t solve any linearized gov eqns any longer, so perhaps adjust to ‘primal and adjoint problems’

Comment (WallyMaier) [reply SU2_CFD/src/solver_structure.cpp]: Done

**Pull request merged**